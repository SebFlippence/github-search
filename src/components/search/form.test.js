import React from 'react';
import { render } from '@testing-library/react';

import SearchForm from './form';

test('renders correctly', () => {
  const { baseElement } = render(<SearchForm search="for a user" />);
  expect(baseElement).toMatchSnapshot();
});
