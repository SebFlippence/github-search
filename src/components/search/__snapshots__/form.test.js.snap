// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<body>
  <div>
    <div
      class="App"
    >
      <div
        class="w-full max-w-md"
      >
        <form
          action=""
          class="px-8 py-8 pt-8"
        >
          <div
            class="px-4 pb-4"
          >
            <label
              class="text-xl block font-bold pb-2"
              for="username"
            >
              Search for a GitHub username
            </label>
            <input
              class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline border-blue-300 "
              id="username"
              name="username"
              placeholder="for a user"
              type="username"
            />
          </div>
          <div
            class="px-4 pb-4"
          >
            <input
              class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline"
              type="submit"
              value="Search"
            />
          </div>
        </form>
      </div>
    </div>
  </div>
</body>
`;
